test/api/review/api_test.go

```diff
@@ -0,0 +1,155 @@
+package review
+
+import (
+	"testing"
+
+	"github.com/konveyor/tackle2-hub/api"
+	"github.com/konveyor/tackle2-hub/test/assert"
+)
+
+func TestReviewCRUD(t *testing.T) {
+	for _, r := range Samples {
+		t.Run("Review CRUD", func(t *testing.T) {
+
+			// Create Application and Review.
+			app := api.Application{
+				Name:        r.Application.Name,
+				Description: "Application for Review",
+			}
+			assert.Must(t, Application.Create(&app))
+
+			// Update Application ID with the Sample Id.
+			r.Application.ID = app.ID
+
+			assert.Must(t, Review.Create(&r))
+
+			// Get.
+			got, err := Review.Get(r.ID)
+			if err != nil {
+				t.Errorf(err.Error())
+			}
+
+			// Compare got values with expected values.
+			AssertEqualReviews(t, got, r)
+
+			// Update Comments and Effort Estimate.
+			r.Comments = "Updated Comment " + r.Comments
+			assert.Should(t, Review.Update(&r))
+
+			// Find Review and check its parameters with the got(On Updation).
+			got, err = Review.Get(r.ID)
+			if err != nil {
+				t.Errorf(err.Error())
+			}
+
+			// Check if the unchanged values remain same or not.
+			AssertEqualReviews(t, got, r)
+
+			// Delete Related Applications.
+			assert.Must(t, Application.Delete(app.ID))
+		})
+
+		t.Run("Delete Review and its dependencies", func(t *testing.T) {
```

Comment:
Not sure why is application delete before the reviews (so why there is a separated subtets for the deletion).

Reply:
Will merge the test!